.gitignore

```diff
@@ -39,3 +39,8 @@ eslint_report.htm
 ### VisualStudioCode Patch ###
 # Ignore all local history of files
 .history
+.idea/inspectionProfiles/Project_Default.xml
+.idea/modules.xml
+.idea/vcs.xml
+.idea/workspace.xml
+.idea/Zettlr.iml
```

Comment:
I understand, but I still need it let git ignore these files. Otherwise I have to delete these files manually every time I make commits. Is there any other way to let git ignore these files? If not, we can delete this when we make PR to the Zettlr repo.

Reply:
One option is to make the Idea changes to your local .gitignore file and not `git add` and not `git commit`. Or, as you wrote, we can roll-back the .gitignore changes becore the Zettlr pull request.